addons/l10n_it_edi/models/account_invoice.py

```diff
@@ -104,13 +110,16 @@ def _l10n_it_edi_prepare_fatturapa_line_details(self, reverse_charge_refund=Fals
                     sep = ', ' if description else ''
                     description = f"{description}{sep}{downpayment_moves_description}"
 
+            split_payment_tax = next((x for x in line.tax_ids.mapped(lambda t: t._l10n_it_get_split_payment_tax()) if x), False)
```

Comment:
line.tax_ids.flatten_taxes_hierarchy()._l10n_it_filter_kind('vat') should do the trick.  

Who says someone will not put another vat tax in a group?  @lordkrandel

Reply:
Ok, I understood, we'll use it also in that function then.